src/decaygen.py

```diff
@@ -224,7 +224,7 @@ def b_from_a(cse, a_i):
     bkey = EXP_EXPR.format(a=a_i)
     return cse[bkey]
 
-def chainexpr(chain, cse, b, short=1e-8):
+def chainexpr(chain, cse, b, bt, short=1e-8):
```

Comment:
this keeps track of the total end output and make sure we are not "creating" mass. This seems to be an issue when we have multiple paths to the same stable final nuclide.